cmd/podman/root.go

```diff
@@ -236,16 +236,12 @@ func loggingHook() {
 
 func rootFlags(cmd *cobra.Command, opts *entities.PodmanConfig) {
 	cfg := opts.Config
+	uri, ident := resolveDestination()
 
 	lFlags := cmd.Flags()
-	custom, _ := config.ReadCustomConfig()
-	defaultURI := custom.Engine.RemoteURI
-	if defaultURI == "" {
-		defaultURI = registry.DefaultAPIAddress()
-	}
 	lFlags.BoolVarP(&opts.Remote, "remote", "r", false, "Access remote Podman service (default false)")
-	lFlags.StringVar(&opts.URI, "url", defaultURI, "URL to access Podman service (CONTAINER_HOST)")
-	lFlags.StringVar(&opts.Identity, "identity", custom.Engine.RemoteIdentity, "path to SSH identity file, (CONTAINER_SSHKEY)")
+	lFlags.StringVar(&opts.URI, "url", uri, "URL to access Podman service (CONTAINER_HOST)")
```

Comment:
I'm feeling pretty squicked out by the interchange of URI and URL. I think that will lead to maintenance headaches. From my (limited) understanding, I believe URL (l as in Lima) is the correct term. Would you consider replacing the code instances of `uri` (i as India) to `url` for consistency?

Reply:
Yes let's stick with UR**L**

Reply:
uri was chosen for a reason. url is a golang package name. There is a lot of code that uses the url package to process the "uri" variable.  We would be forced to alias the package name or make up longer names which I deemed to be a large maintenance headache.  I attempt to use uri for internals and URL for user facing so everyone is happy.

As a url is a uri this naming simplifies the  coding and is correct naming.

Reply:
Would calling it `path` be a good compromise? It's less specific as to what exact type of path it is, and only one character longer.

Reply:
Path is an element of the url/uri, so that would be confusing to see path.Path.

Reply:
How about we call everything URI and drop URL, if we want to still support the --url for backwards compatibility just make it an alias.